### PR TITLE
Implement PaneItemAction

### PR DESCRIPTION
**Pull request by WinXaito**
Implement PaneActionItem (see #104)

## Pre-launch Checklist

- [x] I have run `dartfmt` on all changed files  
- [x] I have updated `CHANGELOG.md` with my changes  
- [x] ~~I have run "optimize/organize imports" on all changed files~~
- [x] ~~I have addressed all analyzer warnings as best I could~~
- [ ] I have added/updated relevant documentation
- [x] I have run `flutter pub publish --dry-run` and addressed any warnings

### CODE REVIEW

**Comment by WinXaito** [comment]
- Need this on the example app ?
- Need documentation ?

**Comment by richard457** [comment]
> * Need this on the example app ?
> * Need documentation ?

Yes, this will help to know a general idea and for quick start.

File: lib/src/controls/navigation/navigation_view/pane_items.dart [hunk 1]
```diff
@@ -282,6 +285,54 @@ class PaneItemHeader extends NavigationPaneItem {
...
+  @override
+  Widget build(
+    BuildContext context,
+    bool selected,
+    VoidCallback? onPressed, {
+    PaneDisplayMode? displayMode,
+    bool showTextOnTop = true,
+    bool? autofocus,
+  }) {
+    return super.build(context, selected, onTap,
```
Comment (NicolaVerbeeck) [inline lib/src/controls/navigation/navigation_view/pane_items.dart:329]: Formatting

Comment (WinXaito) [reply lib/src/controls/navigation/navigation_view/pane_items.dart:329]:
```
dart format lib/src/controls/navigation/navigation_view/pane_items.dart
Formatted 1 file (0 changed) in 0.23 seconds.
```

Compliant

File: lib/src/controls/navigation/navigation_view/pane_items.dart [hunk 2]
```diff
@@ -282,6 +285,54 @@ class PaneItemHeader extends NavigationPaneItem {
...
+    title,
+    infoBadge,
+    focusNode,
+    autofocus = false,
+  }) : super(
+            icon: icon,
+            title: title,
+            infoBadge: infoBadge,
+            focusNode: focusNode,
+            autofocus: autofocus);
```
Comment (bdlukaa) [inline lib/src/controls/navigation/navigation_view/pane_items.dart:315]: trailing comma

File: lib/src/controls/navigation/navigation_view/pane_items.dart [hunk 3]
```diff
@@ -282,6 +285,54 @@ class PaneItemHeader extends NavigationPaneItem {
...
+    bool selected,
+    VoidCallback? onPressed, {
+    PaneDisplayMode? displayMode,
+    bool showTextOnTop = true,
+    bool? autofocus,
+  }) {
+    return super.build(context, selected, onTap,
+        displayMode: displayMode,
+        showTextOnTop: showTextOnTop,
+        autofocus: autofocus);
```
Comment (bdlukaa) [inline lib/src/controls/navigation/navigation_view/pane_items.dart:332]: trailing comma

**Review by bdlukaa — CHANGES_REQUESTED** [review]
Usage on example app would be good as well!

File: lib/src/controls/navigation/navigation_view/pane_items.dart (hunk 2 above)
Comment (WinXaito) [reply lib/src/controls/navigation/navigation_view/pane_items.dart]: Fixed

File: lib/src/controls/navigation/navigation_view/pane_items.dart (hunk 3 above)
Comment (WinXaito) [reply lib/src/controls/navigation/navigation_view/pane_items.dart]: Fixed

File: example/lib/main.dart [hunk 4]
```diff
@@ -237,6 +238,14 @@ class _MyHomePageState extends State<MyHomePage> {
...
             ),
           ),
+          PaneItemSeparator(),
+          PaneItemAction(
+            icon: const Icon(FluentIcons.open_source),
+            title: const Text('Source code'),
+            onTap: () {
+              launch('https://github.com/bdlukaa/fluent_ui');
+            },
+          ),
```
Comment (bdlukaa) [inline example/lib/main.dart:248]: Isn't it better on `footerItems`?

Comment (WinXaito) [reply example/lib/main.dart:248]: I hesitated, honestly I don't know, both seem similar. Do you have a preference?

1. 
image

2. 
image

3. 
image

4. 
image

Comment (bdlukaa) [reply example/lib/main.dart:248]: 2 is the best IMHO

**Review by bdlukaa — APPROVED** [review]

**Pull request merged**